Tests/CreateAPITests/GenerateFeaturesTests.swift

```diff
@@ -5,10 +5,7 @@ final class GenerateFeaturesTests: GenerateTestCase {
     func testQueryParameters() throws {
         try snapshot(
             spec: .testQueryParameters,
-            name: "test-query-parameters",
-            arguments: [
-                "--package", "test-query-parameters"
-            ]
+            name: "test-query-parameters"
```

Comment:
In `GenerateTestCase`'s `snapshot` method, I've updated it to infer the `moduleName` from the `name` parameter. This means that in a lot of places we no longer need to pass the `moduleName` so you'll see that `--package` turns into nothing in the diff.